### ISSUE TITLE
How to train 5-shot model？

### ISSUE THREAD

**Issue by ljjcoder** [icoz69/DeepEMD#5]
Hi, thanks for your great work!  How should I train the 5-shot model? Direct training from scratch or use 1-shot model as initialization? Do relevant parameters such as learning rate need to be adjusted accordingly?

**Comment by icoz69** [comment]
hello, in the original paper, we do not train a 5-shot model. We directly use the trained 1-shot model for eval. You can set the -shot and model_dir in the eval.py to directly test 5-shot. You may also choose to train the 5-shot model, and you only need to set the -shot in the train_meta.py

**Comment by ljjcoder** [comment]
Thanks for your reply

**Comment by ljjcoder** [comment]
@icoz69 sorry to bother you again.
![1594542390(1)](https://user-images.githubusercontent.com/40750189/87242769-4f083b00-c462-11ea-8b25-9c77ae0b40ae.png)
 I noticed that you didn't seem to report the test results of FCN (5x5), sampling (9patch) on 5-shot. I trained the model with your code and then tested it. I got 65.64+0.28 and 67.82+0.27 accuracy on 1-shot. It is very close to you report result(65.91+0.82,68.09). when I test on 5-shot, I got 81.64+0.57 and 83.78+0.53 accuracy. Is this result normal?
